ci/github/phpunit/mysqli.xml

```diff
@@ -1,12 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="../../vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="../../../vendor/phpunit/phpunit/phpunit.xsd"
          colors="true"
          beStrictAboutOutputDuringTests="true"
          beStrictAboutTodoAnnotatedTests="true"
          failOnRisky="true"
+         failOnWarning="true"
+         convertDeprecationsToExceptions="true"
```

Comment:
According to the docs, this one is already `true` by default

Reply:
This has changed recently, see https://github.com/sebastianbergmann/phpunit/blob/9.5/ChangeLog-9.5.md#9510---2021-09-25

Reply:
Ok, let's keep it then.